pjlib-util/include/pjlib-util/sha1.h

```diff
@@ -58,7 +58,7 @@ PJ_DECL(void) pj_sha1_init(pj_sha1_context *ctx);
  */
 PJ_DECL(void) pj_sha1_update(pj_sha1_context *ctx, 
                              const pj_uint8_t *data, 
-                             const pj_size_t nbytes);
+                             pj_size_t nbytes);
```

Comment:
Thank you for the patch. The rest seems okay. I do not understand this, though. Why is this necessary? `nbytes/len` doesn't seem to be overwritten within the function, and if it is, then I'd rather change the implementation and leave the API declaration untouched.

Reply:
Passing an unsigned int/long value does not require const(^_^) .  pj_md5_update() is simillar, no const

Reply:
Here's what I found from Googling:
Declaring a function with const int parameter simply indicates that the value of the integer parameter will not be modified inside the function. It is a common practice in C to declare function parameters as const if the function does not intend to modify the parameter value.

So while it's true that it's not strictly necessary, I don't think it's worth changing as well, especially considering it is an API. Besides, when I tried to compile it here with both `gcc` and `clang` with `-Wextra`, I didn't receive any warning, so I am curious as to what warning did you get there?

Reply:
I think const mean not modify original memory value, here is just pass a int value， not able to afect original value.

Better to keep same with other xx_update() (md5,hmac ...)